util/drakeUtil.h

```diff
@@ -45,6 +46,45 @@ template <typename Derived> inline void destroyDrakeMexPointer(const mxArray* mx
 //  mexPrintf(mexIsLocked() ? "mex file is locked\n" : "mex file is unlocked\n");
 }
 
+template <typename DerivedA>
+mxArray* eigenToMatlab(const DerivedA &m)
```

Comment:
Yes, I know your view on it.  :)  I think it's ok in this case, simply because this function doesn't do any non-trivial sharing between separately compiled objects.

On Feb 7, 2015, at 12:35 PM, Twan Koolen <notifications@github.com> wrote:

> In util/drakeUtil.h:
> 
> > @@ -45,6 +46,45 @@ template <typename Derived> inline void destroyDrakeMexPointer(const mxArray* mx
> >  //  mexPrintf(mexIsLocked() ? "mex file is locked\n" : "mex file is unlocked\n");
> >  }
> >  
> > +template <typename DerivedA>
> > +mxArray* eigenToMatlab(const DerivedA &m)
> Well I suppose you know my opinion on this by now. I can do the explicit instantiation thing again, but it just eats up my time with nothing to show for it in my view.
> 
> —
> Reply to this email directly or view it on GitHub.
>